echopype/utils/prov.py

```diff
@@ -5,6 +5,9 @@
 from _echopype_version import version as ECHOPYPE_VERSION
 from typing_extensions import Literal
 
+# TODO: It'd be cleaner to use PathHint, but it leads to a circular import error
+# from ..core import PathHint
```

Comment:
Do you mean that defining `ProcessType` and doing `from ..core import ProcessType` (instead of importing `PathHint`) here lead to a circular import error?